homeassistant/components/ecobee/climate.py

```diff
@@ -86,7 +87,7 @@
     "auxHeat2": CURRENT_HVAC_HEAT,
     "auxHeat3": CURRENT_HVAC_HEAT,
     "fan": CURRENT_HVAC_FAN,
-    "humidifier": None,
+    "humidifier": CURRENT_HVAC_DRY,
```

Comment:
`CURRENT_HVAC_DRY` means the the device is drying the air. Is that what the `"humidifier"` action does?

Reply:
Humidifiers add moisture to the air, so it should be left at none.